README.md

```diff
@@ -219,13 +219,14 @@ build/credhub login -s https://localhost:9000 --client-name=credhub_client --cli
 
 First, be sure to pull and compile the [credhub-cli](https://github.com/cloudfoundry-incubator/credhub-cli), as described above.
 
-Make sure your development server is running. When it starts up for the first time, it will create a server CA and server certificate for SSL, as well as a trusted client CA for testing mutual TLS authentication. These will be located in `src/test/resources` relative to the `credhub` repository.
+Make sure your development server is running. When it starts up for the first time, it will create a server CA and server certificate for SSL, as well as a trusted client CA for testing mutual TLS authentication. These will be located in `applications/credhub-api/src/test/resources` relative to the `credhub` repository.
 
 Pull [credhub-acceptance-tests](https://github.com/cloudfoundry-incubator/credhub-acceptance-tests) and run:
 
 ```shell
-CREDENTIAL_ROOT=/path/to/credhub/repo/plus/src/test/resources ./scripts/run_tests.sh
+CREDENTIAL_ROOT=/path/to/credhub/repo/plus/applications/credhub-api/src/test/resources ./scripts/run_tests_h2.sh
```

Comment:
This should not be changed as `run-tests.sh` does the test with 3 different databases, including h2.

Reply:
Removed